### ISSUE TITLE
error: cannot find symbol @Generated on Kapt Task

### ISSUE THREAD

**Issue by allenhsu17media** [google/protobuf-gradle-plugin#446]
As title. I need use that way to fix it.

ref : https://github.com/google/dagger/issues/1449

add 

```
compileOnly "javax.annotation:javax.annotation-api:1.3.2"
```
```
kapt { javacOptions { option("--source", "8")
    option("--target", "8") } }
```
Could you have any idea to fix it like dagger solution?

**Comment by voidzcy** [comment]
The issue is not related to this plugin and I do not have experience using Kapt. I'd suggest seeking help on StackOverflow.

**Comment by allenhsu17media** [comment]
No, I mean is the plugin build will show the error from android

`error: cannot find symbol @Generated` 

I think issue like this 
https://github.com/google/auto/blob/master/common/src/main/java/com/google/auto/common/GeneratedAnnotations.java#L54-L61

when you use jdk version that more than the 8, i think it's problem?

**Comment by voidzcy** [comment]
Still that's not a problem caused by this plugin. This plugin does not configure any annotation processor. 

According to https://github.com/google/dagger/issues/1449#issuecomment-516533776, your issue seems to be caused by a kapt bug and it should have been [fixed in Kotlin 1.3.70](https://github.com/google/dagger/issues/1449#issuecomment-595238637). 

Anyway, this is something completely irrelevant and we are not able to provide help.
